tracy-client/src/span.rs

```diff
@@ -184,7 +186,7 @@ impl Drop for Span {
     }
 }
 
-/// Construct a <code>&'static [SpanLocation]</code>.
+/// Construct a `&'static [SpanLocation]`.
```

Comment:
I suspect this may be a change in the generated docs from a link to `SpanLocation` to a `[SpanLocation]` without a link.

Reply:
Ah, yeah you're right. I spaced and read this as an array for whatever reason.